gateway/src/shard/impl.rs

```diff
@@ -289,16 +290,18 @@ impl Information {
     }
 
     /// When the ratelimiter will next refill the [`ratelimit_requests`].
+    /// This will be `None` if payload ratelimiting has been disabled.
```

Comment:
```suggestion
    /// When the ratelimiter will next refill the [`ratelimit_requests`].
    ///
    /// This will be `None` if payload ratelimiting has been disabled.
```